packages/cactus-test-tooling/src/main/typescript/corda/corda-v5-test-ledger.ts

```diff
@@ -0,0 +1,267 @@
+import { EventEmitter } from "events";
+//import path from "path";
+
+import { Config as SshConfig } from "node-ssh";
+import Docker, { Container, ContainerInfo } from "dockerode";
+import Joi from "joi";
+
+import { ITestLedger } from "../i-test-ledger";
+import { Containers } from "../common/containers";
+
+import {
+  LogLevelDesc,
+  Logger,
+  LoggerProvider,
+  Checks,
+  Bools,
+} from "@hyperledger/cactus-common";
+
+/*
+ * Contains options for Corda container
+ */
+export interface ICordaV5TestLedgerConstructorOptions {
+  imageVersion?: string;
+  imageName?: string;
+  publicPort?: number;
+  postgresPort?: number;
+  logLevel?: LogLevelDesc;
+  emitContainerLogs?: boolean;
+  rpcPortA?: number;
+}
+
+/*
+ * Provides default options for Corda container
+ */
+const DEFAULTS = Object.freeze({
+  imageVersion: "latest",
```

Comment:
Instead of latest tag, please specify the exact tag. This is very helpful in reconstructing the error situations later on (Suppose if a CI is failing because of this ledger in April with the latest tag and the tag gets updated in May and if someone tries to fix it in June then the latest tag might not be able to reconstruct the original error scenario)